codegenerator/cli/templates/static/codegen/src/db/TablesStatic.res

```diff
@@ -6,6 +6,21 @@ let isNullable = true
 let isIndex = true
 
 module EventSyncState = {
+  //Used unsafely in DbFunctions.res so just enforcing the naming here
+  let tableName = "event_sync_state"
+  let blockTimestampFieldName = "block_timestamp"
+  let blockNumberFieldName = "block_number"
+  let logIndexFieldName = "log_index"
+  let isPreRegisteringDynamicContractsFieldName = "is_pre_registering_dynamic_contracts"
+
+  //We need to update values here not delet the rows, since restarting without a row
+  //has a different behaviour to restarting with an initialised row with zero values
+  let resetCurrentCurrentSyncStateQuery = `UPDATE public.${tableName}
+    SET ${blockNumberFieldName} = 0, 
+        ${logIndexFieldName} = 0, 
+        ${blockTimestampFieldName} = 0, 
+        ${isPreRegisteringDynamicContractsFieldName} = false;`
```

Comment:
Let's maybe move it below and use values from table. So there's a single source of truth
```suggestion
  let resetCurrentCurrentSyncStateQuery = `UPDATE ${table.schemaName}.${table.tableName}
    SET ${blockNumberFieldName} = 0, 
        ${logIndexFieldName} = 0, 
        ${blockTimestampFieldName} = 0, 
        ${isPreRegisteringDynamicContractsFieldName} = false;`
```